platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/LogoutRequestService.java

```diff
@@ -365,7 +365,7 @@ public Response postLogoutRequest(
     if (encodedSamlRequest != null) {
       try {
         LogoutRequest logoutRequest =
-            logoutMessage.extractSamlLogoutRequest(decodeBase64(encodedSamlRequest));
+            logoutMessage.extractSamlLogoutRequest(RestSecurity.base64Decode(encodedSamlRequest));
```

Comment:
❓ What is the reason for switching to `RestSecurity.base64Decode`?

Reply:
That's because we can reuse the decoding method in RestSecurity (like we do in some places) instead of creating an identical method here